flake8_adjustable_complexity/checker.py

```diff
@@ -45,33 +52,64 @@ class CyclomaticComplexityAjustableChecker:
     name = 'flake8-adjustable-complexity'
     version = version
 
-    _error_message_template = 'CAC001 is too complex ({0} > {1})'
+    _error_message_template = 'CAC001 {0} is too complex ({1} > {2})'
 
     max_mccabe_complexity = 0
+    max_complexity_per_path: Dict[str, int] = {}
 
-    def __init__(self, tree, filename: str):
+    def __init__(self, tree: ast.AST, filename: str):
         self.filename = filename
         self.tree = tree
 
-    @classmethod
-    def add_options(cls, parser) -> None:
+    @ classmethod
+    def add_options(cls, parser: OptionManager) -> None:
+        for option in ('--max-mccabe-complexity', '--max-adjustable-complexity'):
```

Comment:
It seems a bad idea to me to add option alias in this way

Reply:
Can you explain why?